src/runner/settings_window.cpp

```diff
@@ -682,6 +682,8 @@ std::string ESettingsWindowNames_to_string(ESettingsWindowNames value)
         return "Dashboard";
     case ESettingsWindowNames::AdvancedPaste:
         return "AdvancedPaste";
+    case ESettingsWindowNames::NewPlus:
+        return "AdvancedPaste";
```

Comment:
I assume you want a different string here ;)

Reply:
Ahm... yes :-)